### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by josephlanoux**
Update documentation on building the EMA and Docker container.

### What is the purpose of this change?

Clarify documentation steps to install the EMA and add information on creating a Docker container for the EMA.

### How was this change implemented?

Update the documentation.

### How was this change tested?

N/A

### Is there anything the reviewers should focus on/be aware of?

Only review the documentation.

### CODE REVIEW

File: service/application/docker/runEventManagementAgentDocker.sh [hunk 1]
```diff
@@ -6,4 +6,4 @@ if [ -z "$1" ]
     exit 1
 fi
 
-docker run -d -p 8180:8180 --name event-management-agent event-management-agent:$1
\ No newline at end of file
+docker run -d -p 8180:8180 -v /tmp/configFiles/perf1-ema.yml:/config/ema.yml --name event-management-agent event-management-agent:$1
```
Comment (gregmeldrum) [inline service/application/docker/runEventManagementAgentDocker.sh:9]: The config file path and name should be a parameter:

```
#!/usr/bin/env bash

if [ -z "$2" ]
  then
    echo "usage: runEMADocker <tag> <configFile>"
    exit 1
fi

docker run -d -p 8180:8180 -v $2:/config/ema.yml --name event-management-agent event-management-agent:$1

Comment (dennis-brinley) [reply service/application/docker/runEventManagementAgentDocker.sh:9]: Yes, agree. Beat me to it.

File: README.md [hunk 2]
```diff
@@ -51,23 +51,22 @@ On the roadmap:
...
+* Java 11 (AdoptOpenJDK 11.0.14+ https://adoptium.net/temurin/releases)
+* Maven
+* Docker (recommended)
+
+### Event Management Agent connection file
+
+Specific properties are required to run the Event Management Agent in the connected to Event Portal mode. </br>
 These properties are defined in the connection file that can be downloaded from the Event Portal's Event Management Agent definition. </br>
 The steps for generating the connection file are listed over here : https://docs.solace.com/Cloud/Event-Portal/event-portal-collect-runtime-data.htm#creating_connection_file </br>
 Replace the application.yml present in the location : event-management-agent/service/application/src/main/resources with the EMA connection file. </br>
```
Comment (dennis-brinley) [inline README.md:72]: I think that we need to do away with the application.yml compiled into the java jar. It's bad form to compile identifiers w/ security implications into an exec. Executed from docker, this file is passed to executable as a spring boot over-ride. We should recommend the same method when running the agent as a process / command line.

File: README.md [hunk 3]
```diff
@@ -91,88 +90,32 @@ cd event-management-agent/service
 mvn clean install
 ```
 
-3. Start the Event Management Agent
+3. Run the Event Management Agent as a process (recommended for testing and proof of concept purposes only)
 
 ```
 java -jar application/target/event-management-agent-1.0.0-SNAPSHOT.jar
```
Comment (dennis-brinley) [inline README.md:96]: add spring boot config over-ride parameter. e.g.: 
```bash
java -jar application/target/event-management-agent-1.0.0-SNAPSHOT.jar --spring.config.location=configs/AcmeRetail.yml
```

**Review by dennis-brinley — CHANGES_REQUESTED** [review]
- Add parameter in runEventManagementAgentDocker.sh when running ema as docker image to specify config file.

File: README.md (hunk 2 above)
Comment (josephlanoux) [reply README.md:72]: I agree. We need to plan this work.

**Review by dennis-brinley — APPROVED** [review]

**Pull request merged**